client/clienterror/clienterror.go

```diff
@@ -86,6 +87,34 @@ func New(err error) *APIError {
 		return ae
 	}
 
+	createClusterUnauthorizedErr, ok := err.(*clusters.AddClusterUnauthorized)
+	if ok {
+		return &APIError{
+			HTTPStatusCode: http.StatusUnauthorized,
+			OriginalError:  createClusterUnauthorizedErr,
+			ErrorMessage:   "Unauthorized",
+			ErrorDetails:   "You don't have permission to create a cluster for this organization.",
+		}
+	}
+	createClusterDefaultErr, ok := err.(*clusters.AddClusterDefault)
+	if ok {
+		ae := &APIError{
+			HTTPStatusCode: createClusterDefaultErr.Code(),
+			OriginalError:  createClusterDefaultErr,
+		}
+		if ae.HTTPStatusCode == http.StatusNotFound {
+			ae.ErrorMessage = "Organization does not exist"
+			ae.ErrorDetails = "The organization to own the cluster does not exist. Please check the name."
+		} else if ae.HTTPStatusCode == http.StatusBadRequest {
+			ae.ErrorMessage = "Invalid parameters"
+			ae.ErrorDetails = "The cluster cannot be created. Some parameter(s) are considered invalid.\n"
+			ae.ErrorDetails += "Details: " + createClusterDefaultErr.Payload.Message
+		}
+		return ae
+	}
+
+	fmt.Printf("%#v\n", err)
```

Comment:
Is this some debugging ?

Reply:
Yes, removed it in the latest PR (already merged).